package.json

```diff
@@ -20,12 +16,17 @@
     "grunt-shell": "1.3.0",
     "shelljs": "0.3.0"
   },
+  "scripts": {
+    "start": "electron main.js",
+    "test": "electron tests.js",
+    "postinstall": "cd ./node_modules/leveldown && node-gyp rebuild --target=1.4.1 --dist-url=https://atom.io/download/atom-shell"
```

Comment:
I'm getting an error from npm install related to leveldown - is there something I need to do to make this work properly?